app/app_xdg.go

```diff
@@ -20,8 +20,15 @@ import (
 	"fyne.io/fyne/v2/theme"
 )
 
+var currentVariant atomic.Value
```

Comment:
Hmm. I actually have a better idea. Given that `fyne.ThemeVariant` is a uint, you can get cleaner and faster code by just using an `atomic.Uint64` and cast the value to a `fyne.ThemeVariant` where loading and storing.

Reply:
Thanks, done